### PR TITLE
Remove webserver bundle usage

### PR DESCRIPTION
**Pull request by MrYamous**
Remove a webserver bundle usage in documentation from 5.0 according to [his removal in Symfony 5.0](https://github.com/symfony/symfony/blob/master/UPGRADE-5.0.md#webserverbundle)

### CODE REVIEW

File: contributing/code/reproducer.rst [hunk 1]
```diff
@@ -65,8 +65,9 @@ to a route definition. Then, after creating your project:
    of controllers, actions, etc. as in your original application.
 #. Create a small controller and add your routing definition that shows the bug.
 #. Don't create or modify any other file.
-#. Execute ``composer require symfony/web-server-bundle`` and use the ``server:run``
-   command to browse to the new route and see if the bug appears or not.
+#. Install the :doc:`local web server </setup/symfony_server>` provided by Symfony
+   and use the ``symfony server:start`` command to browse to the new route and
```
Comment (HeahDude) [inline contributing/code/reproducer.rst]: `symfony serve`?

**Review by HeahDude — APPROVED** [review]
Thanks for that PR

File: contributing/code/reproducer.rst (hunk 1 above)
Comment (OskarStark) [reply contributing/code/reproducer.rst]: „serve“ is just an alias. „server:start“ is fine 👌🏻

Comment (HeahDude) [reply contributing/code/reproducer.rst]: `server:start` is also an alias :D

Comment (OskarStark) [reply contributing/code/reproducer.rst]: Oh really ? 😊

Comment (javiereguiluz) [reply contributing/code/reproducer.rst]: Both look equally correct to me ... so, to not waste much time discussing about this ... let's merge "as is" 😄

**Comment by javiereguiluz** [comment]
Thank you @MrYamous.

**Pull request merged**